lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -141,6 +164,7 @@ def _span(name: str) -> Generator[Optional[Span], Any, Any]:
         with tracer.start_as_current_span(name) as span:
             yield cast(Span, span)
     else:
+        logger.debug("tracer not found")
```

Comment:
Leftover?

Reply:
Not really: that was on purpose as I spent a day trying to find out that it was the missing tracer contextvar. So in case we find one more bug this line might hopefully lead the poor investigator to a conclusion sooner.

Reply:
Is "tracer not found" a bit too ambiguous?
I wonder if something like this could be useful:
```python
>>> def f():
...     print(" -> ".join(map(lambda v: v.function, inspect.stack())))
... 
>>> # also define g and h
>>> h()
f -> g -> h -> <module>
```